Jenkinsfile

```diff
@@ -128,55 +128,58 @@ def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArg
 }
 
 
-podTemplate(
-    cloud: 'kubernetes-cluster',
-    label: label,
-    idleMinutes: 1,  // Allow some best-effort reuse between successive stages
-    containers: [
-        containerTemplate(
-            name: 'go',
-            image: 'golang:1.10.1-stretch',
-            ttyEnabled: true,
-            command: 'cat',
-            // Rely on burst CPU, but actually need RAM to avoid OOM killer
-            resourceLmitCpu: '2000m',
-            resourceLimitMemory: '2Gi',
-            resourceRequestCpu: '10m',
-            resourceRequestMemory: '1Gi',
-        ),
-        // Note nested podTemplate doesn't work, so use "fat slaves" for now :(
-        // -> https://issues.jenkins-ci.org/browse/JENKINS-42184
-        containerTemplate(
-            name: 'gcloud',
-            image: 'google/cloud-sdk:218.0.0',
-            ttyEnabled: true,
-            command: 'cat',
-            resourceRequestCpu: '1m',
-            resourceRequestMemory: '100Mi',
-            resourceLimitCpu: '100m',
-            resourceLimitMemory: '500Mi',
-        ),
-        containerTemplate(
-            name: 'az',
-            image: 'microsoft/azure-cli:2.0.45',
-            ttyEnabled: true,
-            command: 'cat',
-            resourceRequestCpu: '1m',
-            resourceRequestMemory: '100Mi',
-            resourceLimitCpu: '100m',
-            resourceLimitMemory: '500Mi',
-        ),
-    ],
-    yaml: """
+podTemplate(cloud: 'kubernetes-cluster', label: label, idleMinutes: 1,  yaml: """
 apiVersion: v1
 kind: Pod
 spec:
+  containers:
+  - name: 'go'
+    image: 'golang:1.10.1-stretch'
```

Comment:
Why `gaoling:1.10.1-stretch`? What's this `-stretch` thing?

Reply:
The golang image is available in various flavors. The `-stretch` bit indicates that we want the debian/stretch version

see: https://hub.docker.com/_/golang?tab=tags